Contribute/pull-requests.html

```diff
@@ -7,8 +7,8 @@
 ---
 
 {% capture sub %}
-<p>This is a guide for conducting effective and efficient pull requests on any of
-the seL4 GitHub repositories. Pull requests that do not follow it may not
+<p>This is a guide for conducting effective and efficient pull requests (PRs) on
```

Comment:
We should not introduce abbreviations in the sub-heading, we can't assume people read those.